mlserver/cloudevents.py

```diff
@@ -0,0 +1,61 @@
+import os
+from typing import Dict
+
+CLOUDEVENTS_HEADER_ID = "Ce-Id"
+CLOUDEVENTS_HEADER_SPECVERSION = "Ce-Specversion"
+CLOUDEVENTS_HEADER_SOURCE = "Ce-Source"
+CLOUDEVENTS_HEADER_TYPE = "Ce-Type"
+CLOUDEVENTS_HEADER_SPECVERSION_DEFAULT = "0.3"
+
+CLOUDEVENTS_HEADER_REQUEST_ID = "Ce-Requestid"
+CLOUDEVENTS_HEADER_MODEL_ID = "Ce-Modelid"
+CLOUDEVENTS_HEADER_INFERENCE_SERVICE = "Ce-Inferenceservicename"
+CLOUDEVENTS_HEADER_NAMESPACE = "Ce-Namespace"
+CLOUDEVENTS_HEADER_ENDPOINT = "Ce-Endpoint"
+
+ENV_SDEP_NAME = "SELDON_DEPLOYMENT_ID"
+ENV_PREDICTOR_NAME = "PREDICTOR_ID"
+ENV_MODEL_NAME = "PREDICTIVE_UNIT_ID"
+
+NOT_IMPLEMENTED_STR = "NOTIMPLEMENTED"
```

Comment:
Would it make sense to just use `None` instead of the `"NOTIMPLEMENTED"` string?